air-interpreter/Cargo.toml

```diff
@@ -11,12 +11,12 @@ path = "src/wasm_bindgen.rs"
 
 [[bin]]
 name = "air_interpreter_server"
-path = "src/fce.rs"
+path = "src/marine.rs"
 
 [dependencies]
 air = { path = "../air" }
 
-fluence = { version = "0.6.2", features = ["logger"] }
+fluence = { version = "0.6.5", features = ["logger"] }
 
 # Keep 0.2.65 until this is resolved https://github.com/rustwasm/wasm-pack/issues/886
```

Comment:
isn't https://github.com/rustwasm/wasm-pack/issues/886 resolved ?

Reply:
It is, but it has to be checked and I think it deserves a separate PR.
@coder11 could you please a look?

Reply:
yep